setup.cfg

```diff
@@ -27,11 +27,17 @@ zip_safe = False
 install_requires =
     click>=7.0.0
     click-option-group>=0.5.3
-    invenio-records-marc21[postgresql,elasticsearch7]>=0.1.0
+    invenio-records-marc21[elasticsearch7]>=0.4.0
     requests>=2.0.0
+    # TODO: Prevent build fail remove after it is fixed in base module
+    jinja2<3.1
+    click<=8.0.4
+    Flask<2.1.0
+    Werkzeug<=2.0.3
 
 [options.extras_require]
 tests =
+    wheel
```

Comment:
that is only for local testing or?

Reply:
This line was added in commit #78b7605 by C. Ladurner. I have no clue for what purpose.

Reply:
UPSI!!!!! we should remove it with another commit ;)